src/ai/backend/accelerator/cuda_mock/BUILD

```diff
@@ -0,0 +1,40 @@
+python_sources(
+    name="lib",
+    sources=["**/*.py"],
+    dependencies=[
+        "src/ai/backend/common:lib",
+        ":resources",
+    ],
+)
+
+python_distribution(
+    name="dist",
+    dependencies=[
+        ":lib",
+        "!!stubs/trafaret:stubs",
+    ],
+    provides=python_artifact(
+        name="backend.ai-accelerator-cuda-mock",
+        description="Backend.AI Accelerator Plugin for CUDA (Mockup)",
+        license="Proprietary",
```

Comment:
The mockup plugin should be an open source, like LGPLv3 following other server-side components.